### ISSUE TITLE
Why CoinBlockerList is removed?

### ISSUE THREAD

**Issue by dynamicfire** [StevenBlack/hosts#1104]
It seems like CoinBlockerList got removed, what happened?

**Comment by spirillen** [comment]
Again?? 

But it is related to this issue unfortunately

https://gitlab.com/ZeroDot1/CoinBlockerLists/issues/55

Maybe I should see if I can give a :hand:

**Comment by dynamicfire** [comment]
> Again??
> 
> But it is related to this issue unfortunately
> 
> https://gitlab.com/ZeroDot1/CoinBlockerLists/issues/55
> 
> Maybe I should see if I can give a ✋

Thanks!

**Comment by spirillen** [comment]
Hi @dynamicfire or @StevenBlack  Could you one of you please re-open this issue until further notice for communication to the above ref subject?

**Comment by funilrys** [comment]
@spirillen, Look at the [GitLab issue](https://gitlab.com/ZeroDot1/CoinBlockerLists/issues/55). The CoinBlockerList was removed by Steve @StevenBlack  as a solution to #1080. 

To be more precise, [the following answer](https://gitlab.com/ZeroDot1/CoinBlockerLists/issues/55#note_258376860) from the maintainer "invited" Steve to remove it:

> Hi @StevenBlack The problem is fixed since a while, but I have found that the file was not uploaded correctly due to a network error, I will change this in the next update.

I suppose that the trigger was "in the next update". Steve was probably expecting "ASAP" as the response.

You can still add it as an extension (locally) @dynamicfire.

**Comment by spirillen** [comment]
:thought_balloon:  _**Thought**_ :thought_balloon: 

I this case where the fact is @ZeroDot1 is troubled with his sad ass ISP wouldn't it have been more "_fair_" to temporary add he troubled domain `lib.rus.ec` to the whitelist and keep the [CoinBlockerLists](https://gitlab.com/ZeroDot1/CoinBlockerLists) extension in the data/ ?

The why :question: : All these projects is opensource an voluntary time consuming projects, non of us get a single dime for all our troubles, and this would be a decent simple hand to offer back and at the same time keeping a very good project alive.

What I most concerned about is all those poor kids that will open there new computer from under the tree :christmas_tree: :gift: and fastly turned into mules from the cyber criminals out there.

Please share you thought, as I'm thinking about opening a PR doing this?

Ping @StevenBlack

**Comment by StevenBlack** [comment]
@spirillen CoinBlockerList is removed because its curation has been sloppy and, when clear errors have been  identified and reported, too slow to respond.  Moreover I've seen ample evidence there is little quality control on its pipeline.

I set the bar, here.  CoinBlockerList does not meet my expectations.  It's been a recurring source of problems.  We're done with that source.

> What I most concerned about is all those poor kids that will open there new computer from under the tree 🎄 🎁 and fastly turned into mules from the cyber criminals out there.

@spirillen, c'mon.  That's enough.  Closing.
